toqito/channel_props/is_unitary.py

```diff
@@ -54,7 +54,7 @@ def is_unitary(phi: np.ndarray | list[list[np.ndarray]]) -> bool:
 
     >>> from toqito.channels import depolarizing
     >>> from toqito.channel_props import is_unitary
-    >>> is_unitary(depolarizing(2))
+    >>> is_unitary(depolarizing(2, param_p=1))
```

Comment:
Is there a reason that the parameter is added here instead of using the default as was done before?

Reply:
> Test was failing due to the missing keyword argument

But this doesn't make sense. There is a default value of `0`. 

https://github.com/vprusso/toqito/blob/1379480fc43a9123e141fbd0794eaa8851cb6ec9/toqito/channels/depolarizing.py#L8

Why would this fail, and if so, what was the failure?